src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -343,28 +343,29 @@ public void TestCreateNoWindowProperty(bool value)
         }
 
         [Fact]
-        public void TestWorkingDirectoryProperty()
+        public void TestWorkingDirectoryPropertyInChildProcess()
         {
             CreateDefaultProcess();
```

Comment:
Why are you creating the default process?

Reply:
To check the default case for empty workingDirectory https://github.com/dotnet/corefx/pull/28628/files/dade85e3f9f69a4fbc2d8987550f9b98ff063c14#diff-3b9989a6be55cdb6f928d6ca33ef86f5R351

Reply:
If this is testing a separate thing, it should be in its own test.